vitrina/datasets/forms.py

```diff
@@ -465,6 +468,27 @@ def __init__(self, request=None, organization=None, *args, **kwargs):
             lambda obj: str(obj.translated_label)
         )
 
+    def clean_identifier(self) -> str:
+        identifier = self.cleaned_data.get("identifier")
+        if not identifier:
+            return identifier
+
+        agency = get_object_or_404(Agency, code="risr")
+        if (
+            agency.identifier_validation_type
+            == Agency.IdentifierValidationType.REGEXP
+            and agency.identifier_validation_options
+        ):
```

Comment:
One line statements are considered more readable. To achieve that - extract some parts into variables, like:
```suggestion
        is_regexp = agency.identifier_validation_type
            == Agency.IdentifierValidationType.REGEXP
        if (is_regexp and agency.identifier_validation_options):
```

Reply:
Changed for better readability